kong/endpoint_permission_service_test.go

```diff
@@ -87,3 +88,56 @@ func TestRBACEndpointPermissionservice(T *testing.T) {
 	err = client.Workspaces.Delete(defaultCtx, createdWorkspace.ID)
 	require.NoError(T, err)
 }
+
+func TestRBACEndpointPermissionList(t *testing.T) {
+	RunWhenEnterprise(t, ">=0.33.0", RequiredFeatures{RBAC: true})
+
+	client, err := NewTestClient(nil, nil)
+	require.NoError(t, err)
+	assert.NotNil(t, client)
+
+	pageSize := 1000
```

Comment:
Let's rely on the package-level `pageSize` variable instead of shadowing it here.